cluster/log-dump/log-dump.sh

```diff
@@ -486,7 +489,7 @@ function find_non_logexported_nodes() {
 function dump_nodes_with_logexporter() {
   if [[ -n "${use_custom_instance_list}" ]]; then
     echo "Dumping logs for nodes provided by log_dump_custom_get_instances() function"
-    NODE_NAMES=( $(log_dump_custom_get_instances node) )
+    while IFS='' read -r line; do NODE_NAMES+=("$line"); done < <(log_dump_custom_get_instances node)
```

Comment:
Unless I'm missing where this gets initialized? Follows pattern used by `NON_LOGEXPORTED_NODES` above
```suggestion
    NODE_NAMES=()
    while IFS='' read -r line; do NODE_NAMES+=("$line"); done < <(log_dump_custom_get_instances node)
```

Reply:
Very good suggestion, thanks a lot! Will fix immediately.

Reply:
fixed